### ISSUE TITLE
Readiness check always fails in single mode unless a request has been recently issued

### ISSUE THREAD

**Issue by olepbr** [platbr/kubernetes-health#9]
Hi,

I was playing around with testing this gem locally, using puma in single mode (`PUMA_WORKERS=0`), and noticed that, while liveness probes correctly return 200, readiness probes always return 503, _unless_ I've recently sent a regular request to the application. Enabling logging of puma stats in my application, i noticed that the output of `Puma.stats` looks like this after startup:

```json
{
  "started_at": "2024-11-22T09:50:24Z",
  "backlog": 0,
  "running": 0,
  "pool_capacity": 5,
  "max_threads": 5,
  "requests_count": 0,
  "versions": {
    "puma": "6.4.3",
    "ruby": {
      "engine": "ruby",
      "version": "3.0.6",
      "patchlevel": 216
    }
  }
}
```

Looking at the codebase, I see that `puma_already_started?` does this in single mode: https://github.com/platbr/kubernetes-health/blob/ec041236229cc3c01706b3faebd64bbb0381464f/lib/puma/kubernetes/app.rb#L98-L99

In other words, it assumes that there has to be at least one thread running for the application to be ready, which is not true in single mode, as the default minimum number of threads is 0.

I'm not sure how this should be fixed. It might be that in single mode, if one receives any puma stats at all, that is sufficient to declare readiness? (In other words, the cited `else` branch should just return `true`.)
